packages/integration-tests/src/test-project-setup/create_empty_amplify_project.ts

```diff
@@ -29,5 +35,24 @@ export const createEmptyAmplifyProject = async (
 
   await setupDirAsEsmModule(projectAmplifyDir);
 
+  await execa(
+    packageManagerProps?.executable ?? 'npm',
```

Comment:
If this is going to be a repeated pattern, we may need a testing PackageManagerController abstraction. It would be similar to the one in `create-amplify` but hopefully more lightweight to cover the installing that we need to do in tests

Reply:
+1.

Other feedback is:
Do we need to "install" these or is there `npm link` equivalent we could use ? Or perhaps this was working because we were under single mono repo so packages were resolved automagically ?
Perhaps this is right approach but I wonder why we didn't do this before.

Btw.
If this file becomes sibling of `create-amplify` perhaps we should just run `create-amplify` and then substitute sources of generated project?

Reply:
It was working before because the test directories were created in the project repo so "automagic resolution" happened. Personally I'm not opposed to keeping it that way so that we don't waste test time installing the same packages over and over